### PR TITLE
Added Operations

### LINKED ISSUE #750: Add Operations
image

### PR DESCRIPTION
**Pull request by Fuyuki325**
AddOperations1
AddOperations2

### CODE REVIEW

**Comment by Fuyuki325** [comment]
This fixes #750

File: src/data/forms/Admins.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const AFFILIATIONS = [
   "recruitment",
   "software",
   "UIUX",
+  "Operations",
```
Comment (rfairooz) [inline src/data/forms/Admins.js]: great, just make it lower case so it's consistent with the other options

**Comment by rfairooz** [comment]
> This fixes #750

So for pull requests, instead of commenting the issue, Go under "Development" on the left and choose the issue from the list. This will link your issue to this PR so when we merge that issue will be automatically closed. Good job!

**Review by rfairooz — COMMENTED** [review]
Oh and at the very bottom check if your branch has merge conflicts with the dev branch. If so, fix.

**Comment by Fuyuki325** [comment]
AddOperations3
AddOperations4


fixes Add Operations #750

File: src/data/dynamic/forms/Committees.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ export const AFFILIATIONS = [
   "recruitment",
   "software",
   "UIUX",
+  "Operations",
```
Comment (rfairooz) [inline src/data/dynamic/forms/Committees.js]: make the operations lower case for committee

Comment (shahdivyank) [inline src/data/dynamic/forms/Committees.js]:
```suggestion
  "operations",
```

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**